docs/abcd_rooparametrichist_tutorial/rooparametrichist_exercise.md

```diff
@@ -0,0 +1,507 @@
+----------------------------------------------------------------------------------------------------------------------------------
+BkgRate                 lnN     -         -         -         -         -         -         -         1.05    
+lumi                    lnN     1.016     -         1.016     -         1.016     -         1.016     -
+
+
+```
+</details>
+
+
+
+## Run Fit
+<a id="fit"></a>
+
+From the combined datacard for all the regions, one can run the usual fit diagnostics as follows:
+
+```
+combine -M FitDiagnostics combined_mPhi_1500_2018.txt -m 1500 --saveShapes --saveWithUncertainties --saveNormalizations
+
+```
+
+<details>
+<summary> Results FitDiagnostics  </summary>
+
+```
+ --- FitDiagnostics ---
+Best fit r: 2.4196e-05  -2.4196e-05/+0.276209  (68% CL)
+
+```
+</details>
+
+
+Using the output ```fitDiagnosticsTest.mH1500.root```, one can run the script ```$CMSSW_BASE/src/HiggsAnalysis/CombinedLimit/test/mlfitNormsToText.py``` to get the predictions for the normalizations:
+
+```
+python3 $CMSSW_BASE/src/HiggsAnalysis/CombinedLimit/test/mlfitNormsToText.py fitDiagnosticsTest.mH1500.root
+
+```
+
+<details>
+<summary> Normalizations predictions </summary>
+  
+```
+Channel                                  Process                                Pre-fit              S+B Fit           B-Only Fit
+---------------------------------------------------------------------------------------------------------------------------------
+ch1                                      mPhi_1500                              594.749                0.014                0.000
+ch2                                      mPhi_1500                              605.251                0.015                0.000
+ch3                                      mPhi_1500                              245.161                0.006                0.000
+ch4                                      mPhi_1500                             1541.786                0.037                0.000
+ch1                                      Bkg                                 236222.924           236226.197           236225.436
+ch2                                      Bkg                                 235703.859           235707.150           235706.383
+ch3                                      Bkg                                 381479.390           381476.382           381476.614
+ch4                                      Bkg                                 145963.042           146873.518           146873.251
+
+```
+</details>
+
+Moreover, you can run the script in ```$CMSSW_BASE/src/HiggsAnalysis/CombinedLimit/data/tutorials/longexercise/postFitPlot.py``` to get pre-fit and post-fit plots in the signal region (in the combined datacard ```ch4```):
+
+```
+python3 $CMSSW_BASE/src/HiggsAnalysis/CombinedLimit/data/tutorials/longexercise/postFitPlot.py --input_file fitDiagnosticsTest.mH1500.root --shape_type <shapes_type> --region <region>
```

Comment:
Same filename issue as above.

Reply:
now changed, i was running with a slightly older version of combine where the mass was added in the name of the file